.github/workflows/ci.yml

```diff
@@ -148,3 +150,32 @@ jobs:
           disable_safe_directory: true
           files: lcov.info
           token: ${{ secrets.CODECOV_TOKEN }}
+
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    name: Create new release
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            flags: --target x86_64-unknown-linux-gnu
+            asset_name: "uhyve-${{ github.ref_name }}-linux-amd64"
```

Comment:
Could you put the binaries into archives instead of uploading the binary directly (zip on Windows and .tar.gz on Linux) to avoid warnings when downloading by browser?

See https://github.com/mkroening/edu-sync/blob/1a70834a82c69ed284b2fb620387380c1ef6704f/.github/workflows/release.yml#L35-L43 for example.